odbc/src/api/definitions.rs

```diff
@@ -1,20 +1,23 @@
 use num_derive::FromPrimitive;
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq, FromPrimitive)]
+#[repr(i32)]
 pub enum SqlBool {
     False = 0,
     True,
 }
 
 // Environment attributes
 #[derive(Copy, Clone, Debug, PartialEq, Eq, FromPrimitive)]
+#[repr(i32)]
```

Comment:
Not that it matter size wise, but in the ODBC headers, the values are all unsigned long. Maybe to match, we should map it to u32 (the spec says i32, but the code in this case seems a better of truth to me)?

Reply:
Agreed, the headers are safer to follow.